projects/lights_control/src/main.c

```diff
@@ -3,19 +3,109 @@
 #include "can.h"
 #include "gpio.h"
 #include "interrupt.h"
+#include "log.h"
 #include "status.h"
 #include "wait.h"
 
+#include "lights_blinker.h"
+#include "lights_board_type.h"
+#include "lights_can.h"
+#include "lights_can_config.h"
+#include "lights_gpio.h"
+#include "lights_gpio_config.h"
+#include "lights_gpio_config_front.h"
+#include "lights_signal_fsm.h"
+#include "lights_strobe.h"
+
+#define LIGHTS_SIGNAL_BLINKER_DURATION 500
+#define LIGHTS_SIGNAL_SYNC_COUNT 5
+#define LIGHTS_STROBE_BLINKER_DURATION 500
+
+static LightsSignalFsm s_signal_fsm = { 0 };
+
+static LightsCanStorage s_lights_can_storage = { 0 };
+
+static LightsStrobeStorage s_lights_strobe = { 0 };
+
+static const GPIOAddress s_board_type_address = { .port = GPIO_PORT_B, .pin = 13 };
+
 int main(void) {
-  Event e;
+  CANSettings can_settings = {
+    .bitrate = CAN_HW_BITRATE_500KBPS,
+    .rx_event = LIGHTS_EVENT_CAN_RX,
+    // clang-format off
+    .tx = {
```

Comment:
Why turn off clang-format for these?

Reply:
I hate clang-format. it sucks

Reply:
I'm gonna disable it everywhere

Reply:
jks I probably forgot to remove it? I removed them now...